python/tests/io/test_io_base.py

```diff
@@ -64,9 +64,9 @@ def exists(self):
 
     def open(self) -> InputStream:
         input_file = open(self.parsed_location.path, "rb")
-        if not isinstance(input_file, InputStream):
-            raise TypeError("Object returned from LocalInputFile.open() does not match the OutputStream protocol.")
-        return input_file
+        # if not isinstance(input_file, InputStream):
```

Comment:
Same here, these can be removed right?

Reply:
We actually need this line for mypy to assert that it is an outputstream. I now commented it out because it would never pass. By reinstating these lines, we can remove the `type: ignore` below

Reply:
Ah I see, ok I missed that.